.github/workflows/create-version-branch.yml

```diff
@@ -0,0 +1,39 @@
+name: Check the Metabase SDK version and create a version branch
+
+on:
+  schedule:
+    - cron: '0 * * * *'  # every hour
+  workflow_dispatch:
+
+jobs:
+  update_release_branch:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get TARGET_BRANCH from latest stable major version of @metabase/embedding-sdk-react
+        id: get-target-branch
+        run: |
+          LATEST_VERSION=$(npm view @metabase/embedding-sdk-react version)
```

Comment:
Looks good, I tried `npm view @metabase/embedding-sdk-react version` and it returns `0.53.10` 👍🏻